lib/pager_api/pagination/kaminari.rb

```diff
@@ -13,8 +13,7 @@ def paginate(*args)
         options[:meta] = meta(paginated_collection, options) if PagerApi.include_pagination_on_meta?
 
         pagination_headers(paginated_collection) if PagerApi.include_pagination_headers?
-
-        render options
+        render json: options
```

Comment:
there is a problem with `render options`, to solve that we could use `return options` and in the controller we can use it like this

```ruby
books = paginate Book.all
render json: books
```

or we could use `render json: options` and 
```ruby
books = Book.all
paginate books
```
as mentioned in the doc

so i chose the second solution to keep the the gem api intact